openshift-knative-operator/hack/007-networkpolicy-mesh.patch

```diff
@@ -22,15 +22,15 @@ index 00000000..3d46fbed
 +apiVersion: networking.k8s.io/v1
 +kind: NetworkPolicy
 +metadata:
-+  name: istio-webhook
```

Comment:
Note for myself. This old networkpolicy also needs to be deleted by operator :sweat: